ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -134,7 +136,28 @@ Delete Pipeline Run
     Navigate To Page    Data Science Pipelines    Runs
     Wait Until Page Contains Element    xpath://span[text()='Triggered']
     Click Element    //span[text()='Triggered']
-    Pipelines.Click Action From Actions Menu    ${pipeline_name}    Delete
+    Pipelines.Click Action From Actions Menu    ${run_name}    Delete
     Handle Deletion Confirmation Modal    ${run_name}    triggered run
     Wait Until Page Contains Element    xpath://h2[contains(text(), 'No triggered runs yet')]
-    Capture Page Screenshot
+
+Delete Pipeline
+    [Documentation]    Delete a pipeline. From the left menu select "Data Science Pipelines" -> Pipelines.
+    ...                The "Delete Pipeline" will search for a line in the grid that match the pipeline name.
+    ...                Based on that, hit the ... Menu in the row and hit Delete drop down menu.
+    [Arguments]    ${pipeline_name}
+    Navigate To Page    Data Science Pipelines    Pipelines
+    Wait Until Page Contains Element    xpath://a[text()='${pipeline_name}']
+    Pipelines.Click Action From Actions Menu    ${pipeline_name}    Delete
+    Handle Deletion Confirmation Modal    ${pipeline_name}    pipeline
+    Wait Until Page Contains Element    xpath://h4[contains(text(), 'No pipelines yet')]
+
+Delete Data Science Project
```

Comment:
This keyword already exists in ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

Reply:
I don't think that adding `Pipelines` or `Workbenches` in `Projects.resource` is the right way.
What should I do?
![image](https://github.com/red-hat-data-services/ods-ci/assets/687311/c3d5a015-6f9d-4eea-9bbe-31ae1f9be6d1)

Reply:
hmm well, from a technical pov, there is nothing wrong in adding "Workbenches.<keyword>" in the Projects.resource. However I would agree it doesn't sound right from a logical pov. 

I think there are 2 solutions:
1. move `Click Action From Actions Menu`(from `Workbenches.resource`) to a common resource file (e.g., `ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Common.resource`). I didn't do this because this one I think was the only common keyword I identified. But I personally have nothing against this solution
2. replace `Delete Data Science Project` with `Delete Data Science Projects From CLI`. It does not use the UI to delete the project.

Reply:
Ok. The `Projects.resource` has `Workbenches.resource`
Refactoring is out of the scope of this PR.
https://github.com/red-hat-data-services/ods-ci/issues/844